views/plugin/xyz/xyzproviderhelp.html

```diff
@@ -0,0 +1,28 @@
+<div id="time-help" class="flex-fill w-100 d-flex flex-column">
+  <div class="modal-body container flex-fill">
+    Please provide a valid XYZ tile layer URL.
+    <br>
+    <br>
+    An XYZ tile layer URL can have many different formats, such as:
+    <ul>
+      <li>https://osm.gs.mil/tiles/default/{z}/{x}/{y}.png</li>
```

Comment:
Example URL's should either be fake (ie, `https://example.com/{z}/{x}/{y}`) or a public facing service like those used in the default opensphere config:
```
https://services.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer/tile/{z}/{y}/{x}
```